certification/internal/engine/engine.go

```diff
@@ -279,3 +298,126 @@ func untar(dst string, r io.Reader) error {
 		}
 	}
 }
+
+func writeCertImage(img v1.Image) error {
+	config, err := img.ConfigFile()
+	if err != nil {
+		return fmt.Errorf("%w: %s", errors.ErrImageInspectFailed, err)
+	}
+
+	digest, _ := img.Digest()
+	manifest, _ := img.Manifest()
+	rawConfig, _ := img.RawConfigFile()
+	size, _ := img.Size()
+	layers, _ := img.Layers()
```

Comment:
Are we ignoring the errors?
Also, I think we can refactor this into a single method and use `reflect` library to call methods dynamically. This seems to be a good example:
https://www.geeksforgeeks.org/reflect-methodbyname-function-in-golang-with-examples/

Reply:
Manifest() and Layers() errors should not be ignored. It'd be safe to ignore RawConfigFile(), Digest() and Size() (they just return nil no matter what).